### DISCUSSION TITLE
create a demo app using next.js and publish to vercel and make it easy just to copy and paste code

### DISCUSSION THREAD

**Discussion by tgmarinho** [mercadopago/sdk-react#84] [General]
create a demo app using next.js and publish to vercel and make it easy just to copy and paste code

**Comment by AleFossati** [comment]
Hello @tgmarinho, how are you?
We do have plans to create more demo apps to make it easy to understand the integration.
In the meantime, this repo also works as a demo and got snippets that you can use to copy/paste.
Have you tried our storybook? Let me know what are you missing in our examples, so we can add more use cases.
